v2/internal/testcommon/samples_tester.go

```diff
@@ -69,6 +69,9 @@ var exclusions = []string{
 
 	// Excluding cdn secret as it requires KV secrets
 	"secret",
+
+	// Exclude backupvaultsbackupinstance as it requires role assignments to be created after backup instance is created to make it land into protection configured state.
```

Comment:
Is this still true? 

Your test in `crd_dataprotection_backupinstance_test.go` creates everything all at once, including the Role Assignment.

Reply:
I think yes, since in samples test suite, refs apply before the actual resources. Currently, we don't apply resources all at once. We have a tech debt tracking(#3091) this which I should be adding in into 2.7.0.

Maybe add a comment and mention the issue number?

Reply:
added issue number in the comment